src/components/JaegerIntegration/JaegerResults/SpanTable.tsx

```diff
@@ -26,6 +26,8 @@ interface SpanDetailState {
   rows: any;
 }
 
+const whiteListIstioSystem = ['istio-ingressgateway', 'jaeger-query'];
```

Comment:
Mmm, this is not right, it should come from the config server backend instead and configurable from the CR as well.

Reply:
ok let @hhovsepy if this fix the problem with jaeger-query and we can think about hot to deal with this, because this only happens with the services of the istio-system namespace. If jaeger is in another namespace it should run without problem